hledger/Hledger/Cli/Commands/Diff.md

```diff
@@ -0,0 +1,21 @@
+diff\
+Compares two journal files.  It looks at the transactions of a single
+account and prints out the transactions which are in one journal file but not
+in the other.  This is particularly useful for reconciling existing journals
+with bank statements.
```

Comment:
This motivating use case is helpful. Could you add a little more concrete detail here ? Eg do you convert a chunk of CSV between the dates of two bank statements to a journal file, and then diff that with your main file ?